smarttub/api.py

```diff
@@ -465,6 +465,7 @@ def __init__(self, spa: Spa, **properties):
 
         self.intensity = properties["intensity"]
         self.mode = self.LightMode[properties["mode"]]
+        self.cycleSpeed = properties["cycleSpeed"]
```

Comment:
It looks like this property is present in my case as well, though I'm not sure if it's always present. To be on the safe side, we should probably handle it being absent